src/number_theory/pythagorean_triples.lean

```diff
@@ -27,10 +27,13 @@ open_locale classical
 /-- Three integers `x`, `y`, and `z` form a Pythagorean triple if `x * x + y * y = z * z`. -/
 def pythagorean_triple (x y z : ℤ) : Prop := x * x + y * y = z * z
 
+/-- Pythagorean triples are interchangable, i.e `x * x + y * y = y * y + x * x = z * z`.
+This comes from additive commutativity -/
 lemma pythagorean_triple_comm {x y z : ℤ} :
  (pythagorean_triple x y z) ↔ (pythagorean_triple y x z) :=
 by { delta pythagorean_triple, rw add_comm }
 
+/-- The zeroth pythagorean triple is zero -/
```

Comment:
```suggestion
/-- The zeroth pythagorean triple is all zeros -/
```

or maybe "three zeros". Either way it should be obvious, since it's a "triple"

Reply:
Whatever flows better.